RELEASE.md

```diff
@@ -0,0 +1,40 @@
+This document walks through the release process for helm-operator-plugins.
+
+1. Creation of pre-release PR:
+
+Update the release version of the plugin at the following locations in the file - `pkg/plugins/hybrid/v1alpha/scaffolds/init.go`. As an example, let us assume that we are about to update the release version from `v0.0.9` to `v0.0.10`. To do so, modify the `hybridOperatorVersion` and `helmPluginVersion` variables to `v0.0.10`.
+
+```diff
+-	hybridOperatorVersion = "0.0.9"
++ 	hybridOperatorVersion = "0.0.10"
+
+ 	// helmPluginVersion is the operator-framework/helm-operator-plugin version to be used in the project
+- 	helmPluginVersion = "v0.0.9"
++ 	helmPluginVersion = "v0.0.10"
+```
+
+Run `make generate` to re-generate the `testdata/` with above modifications.
+
+**Note**
+The release PR with the above mentioned change will **fail** the `sanity` test and needs to be merged forcefully. This step is done because the packages which are to be imported in the scaffolded project and the plugin code reside in the same repository. This will ensure that after the release, the plugin will also import the updated version of library helpers.
+
+2. Create a release branch with the name vX.Y.x. Example: `git checkout -b v0.0.10`.
+
+3. Tag the release commit and verify the generated tag.
+
+```
+export VER=v0.0.10
```

Comment:
```suggestion
VER=v0.0.10
```
No need to pollute the environment by exporting a variable that is only interpreted by the current shell.

Reply:
Also, if you provided this variable setting as "step 0" it would be straightforward to reference it also in the commands in following steps rather than the ugly `<release-branch>` placeholder which is not copy-pasteable.